Sound/Tidal/Pattern.hs

```diff
@@ -4,7 +4,7 @@
 module Sound.Tidal.Pattern where
 
 import Control.Applicative
--- import Data.Monoid
+import Data.Monoid (mappend, mempty, Monoid())
```

Comment:
On second thought, these are all not needed: you can simply write `import Data.Monoid ()`